ui_framework/src/global_styling/utilities/_typography.scss

```diff
@@ -0,0 +1,7 @@
+.uTextAlignRight {
```

Comment:
How about `.kuiForceTextAlignRight`? That way it retains the kui prefix and it is really clear that it's an override.